script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_1.txt

```diff
@@ -66,7 +66,7 @@ Even though she is perfectly capable of performing her duties as a servant, it f
 It's my fault, I'm still not used to it."
 <Page20>
 When I get out of bed, I realize that I have been dressed in my pajamas.
-...If I remember correctly, last night I changed into my school uniform because of the howling... and after I came back, I thought I collapsed straight into bed...?
+...If I remember correctly, last night I changed into my school uniform because of the howling... and after I came back, I thought I collapsed straight onto bed...?
```

Comment:
Straight onto bed is not a phrase people use. I would revert this to into or use `onto my`

Reply:
revert